### PR TITLE
fix: update readme

### PR DESCRIPTION
**Pull request by clindeman**
Replace all anaconda references with miniforge. Change hand drawn red circles with straight red lines in screenshots.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,54 +1,39 @@
...
-
-1. Download the repository from: https://github.com/nhrap-dev/Environment-Setup
-
-2. Unzip the downloaded folder and run the python script install-miniforge.py
+The Hazus Export tool requires Hazus, ArcGIS Desktop, and conda to be installed on your computer. If you haven't already, follow this [README](https://github.com/nhrap-dev/Environment-Setup/tree/update-README) to setup your machine with miniforge and conda before trying to run Export.
 
 
 ## Documentation
 
 For information about the fields, values and units found in exported Hazus results data, please refer to the guide sheets in the data-dictionaries folder: "EQDataDictionary", "TSDataDictionary", "FLDataDictionary", and "HUDataDictionary". https://github.com/nhrap-hazus/export/tree/master/data-dictionaries
```
Comment (be-cunning) [inline README.md]: It is a best practice to use an active tone (instead of passive) when writing software/technical documentation.  Here are 2 great sources on that: 

- https://docs.openstack.org/doc-contrib-guide/writing-style/general-writing-guidelines.html
- https://developers.google.com/tech-writing/one/active-voice

Change this line to something like the following that uses an active tone:

_"Please refer to the guide sheets in the data-dictionaries folder for information about the fields, values, and units found in exported HAZUS results data."_

File: README.md [hunk 2]
```diff
@@ -1,54 +1,39 @@
...
-
 ![Download Export](Images/Step1.png "Download Export")
 
-**2. Double-click "hazus-export-tool.py". If you don't have the Hazus Python Library installed, follow the prompt to install, then double-click "hazus-export-tool.py" again**
-
+**2. Double-click "hazus-export-tool.py".**
 ![Open Export](Images/Step2.png "Open Export")
 
 **3. Select a study region to export from those stored in your dekstop Hazus database, as shown in the drop-down menu.**
-
```
Comment (be-cunning) [inline README.md:39]: There's a typo for desktop here.

Also, this line is a bit wordy. It would be best simplified to something like the following:

_"Select a study region from the drop-down list."_

File: README.md [hunk 3]
```diff
@@ -185,3 +170,9 @@ If you want to export only the reports, then it would need to look like this:
 **Script Failure:**
 
 If the script fails without dropping the bk_* (where * is the name of the hpr file or its .bk sql server backup database) database, then you will need to do it manually. Using SQL Server Management Studio you can DELETE the bk_* database and using Windows explorer you can delete the temp folder that contains the unzipped contents of the HPR file.
+
+## Contact
+
+Issues can be reported through the repository on Github (https://github.com/nhrap-hazus/export)
```
Comment (be-cunning) [inline README.md]: Change this to more active tone. Also it's a best practice to capitalize the 'H' in GitHub.  Below is an example:

_"Please report issues through the Export GitHub repository: https://github.com/nhrap-hazus/export"_

File: README.md [hunk 4]
```diff
@@ -185,3 +170,9 @@ If you want to export only the reports, then it would need to look like this:
...
+
+For questions contact fema-hazus-support@fema.dhs.gov
```
Comment (be-cunning) [inline README.md]: Minor punctuation nit: throw a semicolon after questions.... it should read:

 _"For questions: contact fema-hazus-support@fema.dhs.gov"_

**Review by be-cunning — CHANGES_REQUESTED** [review]
Please make changes per feedback. Also, updates/commits like this are generally a chore, instead of a fix.  Fix is reserved for bug fixes. It's not an issue here, but a good practice to follow in the future.

File: README.md (hunk 4 above)
Comment (clindeman) [reply README.md]: Added colon

File: README.md (hunk 3 above)
Comment (clindeman) [reply README.md]: Changed to active tone as suggested.

File: README.md (hunk 2 above)
Comment (clindeman) [reply README.md:39]: Removed the extra words including the typo.

File: README.md (hunk 1 above)
Comment (clindeman) [reply README.md]: Changed to active tone as suggested.
